src/taskgraph/run-task/run-task

```diff
@@ -1081,6 +1081,12 @@ def main(args):
         b"setup",
         b"run-task started in %s\n" % os.environ["TASK_WORKDIR"].encode("utf-8"),
     )
+    #logging command which envoked the script
```

Comment:
nit: I don't think this comment is necessary, the code is pretty self-explanatory.